proto/proto/penumbra/client/v1alpha1/client.proto

```diff
@@ -25,6 +25,7 @@ import "tendermint/types/validator.proto";
 service ObliviousQueryService {
   rpc CompactBlockRange(CompactBlockRangeRequest) returns (stream CompactBlockRangeResponse);
   rpc ChainParameters(ChainParametersRequest) returns (ChainParametersResponse);
+  rpc Epoch(EpochRequest) returns (EpochResponse);
```

Comment:
I think we might want this to be
```
rpc EpochByHeight(EpochByHeightRequest) returns (EpochByHeightResponse);
```
in case we later want to query for the start/end or other epoch data by some otehr means (e.g. epoch data by index)